docs/source/chapters/about.rst

```diff
@@ -47,22 +46,20 @@ Some of the pgRouting library core features are:
 * `Johnson's Algorithm <https://en.wikipedia.org/wiki/Johnson's_algorithm>`__
 * `Floyd-Warshall Algorithm
   <https://en.wikipedia.org/wiki/Floyd%E2%80%93Warshall_algorithm>`__
-* `A* Algorithm <https://en.wikipedia.org/wiki/A*_search_algorithm>`__
-* `Bi-directional Algorithms
-  <https://en.wikipedia.org/wiki/Bidirectional_search>`__
-  * Bi-directional Dijkstra
-  * Bi-directional A*
-* `Traveling Sales Person
+* `A* Search Algorithm <https://en.wikipedia.org/wiki/A*_search_algorithm>`__
+* `Bi-directional Search Algorithms <https://en.wikipedia.org/wiki/Bidirectional_search>`__
```

Comment:
remove this group

Reply:
By group I mean the Bi-directional Search Algorithms line with link to wikipedia

Reply:
Okay. I have removed it.